api/tests/test_floor_numbers_api.py

```diff
@@ -0,0 +1,107 @@
+from unittest.mock import patch
+from rest_framework.test import APIClient
+from rest_framework.reverse import reverse
+
+from core.models import OfficeBlock, User, FloorNumber
+
+from api.tests import APIBaseTestCase
+
+client = APIClient()
+
+
+class FloorNumberAPITest(APIBaseTestCase):
+    """ Tests for the FloorNumber endpoint"""
+
+    def setUp(self):
+        super(FloorNumberAPITest, self).setUp()
+        self.admin = User.objects.create_superuser(
+            email='testuser@gmail.com', cohort=19,
+            slack_handle='tester', password='qwerty123'
+        )
+
+        self.building = OfficeBlock.objects.create(
+            name="Block A"
+        )
+        self.floor_number = FloorNumber.objects.create(
+            floor_number=5,
+            office_block=self.building
+        )
+        self.floor_number_url = reverse('floor-numbers-list')
+        self.token_user = 'testtoken'
+
+    def test_non_authenticated_user_get_office_block(self):
+        response = client.get(self.floor_number_url)
+        self.assertEqual(response.data, {
+            'detail': 'Authentication credentials were not provided.'
+        })
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_can_post_floor_number(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.admin.email}
+        data = {
+            "floor_number": 25,
+            "office_block": self.building.id
+        }
+        response = client.post(
+            self.floor_number_url,
+            data=data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
+        print(response.data, response.status_code)
+        # self.assertIn("floor_number", response.data.keys())
```

Comment:
Hey, the actual tests are commented out 🙂

Reply:
Fixed.